modules/test/playwright/pages/document-library-web/gogoShell.page.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {Locator, Page} from '@playwright/test';
+
+import {ApplicationsMenuPage} from '../product-navigation-applications-menu/ApplicationsMenuPage';
+export class GogoShellPage {
```

Comment:
Only one comment about the organization of this page. (mybe was talked about so sorry it this was)

IMHO this maybe is better placed in `test/playwright/pages/product-navigation-applications-menu`

And instead of do

`documentLibraryPage.addGogoShellCommand()`
be imported on the **test** and do the call
`gogoShellPage.addCommand(command).`

Reply:
@boton also pointed this. We are not sure where they mustbe placed, so one option was to leave it here and in the mean while ask in the slack channel if they fit there. They have been moved in document library fixtures.